deploy/helm/lunettes/templates/grafana/grafana-cm.yaml

```diff
@@ -696,7 +697,7 @@ data:
                       {
                         "targetBlank": true,
                         "title": "NodeYaml",
-                        "url": "http://lunettes.lunettes.svc.cluster.local:8080/api/v1/debugpodyaml?name=${__data.fields[\"NodeName\"]}"
+                        "url": "http://localhost:9097/d/nodeyamllinks/nodeyaml?orgId=1&var-nodeName=${__data.fields[\"NodeName\"]}"
```

Comment:
Don't use hard-coded jump links. Perhaps you could use the url's hostname variable as the hostname of the jump link

Reply:
We have recorded an issue and will resolve it later

Reply:
we could use the same URL for grafana dashboard access and for node yaml/pod yaml link redirection